### DISCUSSION TITLE
首页的动态横条中的动态数量能不设置上限吗？

### DISCUSSION THREAD

**Discussion by crasher32** [the1812/Bilibili-Evolved#2140] [功能建议 / Ideas]
每次习惯在首页的横条里翻动态
但是翻个几页就断了，还得到全部动态里重新翻
如果横条里的动态数量没有上限，滑到底了自动获取更早的动态就好了

**Comment by RingingVolcano** [comment]
![_YJXCTJ ANXJ6ZP4$3{F@}D](https://user-images.githubusercontent.com/72140353/129303979-fb2ab1d4-5fde-4c88-b35b-53eeb5b91ca7.png)
可以试试开自定义顶栏 这里的动态是无限下拉的

**Reply by crasher32** [reply]
感谢，不过还是希望横幅也能做到无限下拉，毕竟横幅里的动态大一些[]~（￣▽￣）~*

**Reply by RingingVolcano** [reply]
那为啥不直接点进全部动态刷投稿视频呢

**Reply by crasher32** [reply]
哦哦，我现在才看到动态页也有分投稿视频和专栏
以前用首页的动态横条纯粹是因为刷视频方便（┬＿┬）
感谢回复~
